stack.yaml

```diff
@@ -2,7 +2,10 @@ nix:
   packages:
     - zlib
 
-resolver: lts-10.0
+resolver: lts-19.6
 extra-deps:
-- dom-parser-3.0.0
-- generic-arbitrary-0.1.0
+- git: https://github.com/typeable/dom-parser.git
+  commit: 7d556ba528bfa216e99714f8a0d4d00f452803a9
+
+- git: https://github.com/fumieval/xml-lens.git
+  commit: 4612e5ea18822baef678d7176c1ab5f93e4e31e9
```

Comment:
maybe we should release them on hackage instead?

Reply:
I've got them from our projects. Not sure if they are needed really here. I will check.
(Anyway we can release only dom-parser I suppose)